holidays/locale/en_US/LC_MESSAGES/IN.po

```diff
@@ -0,0 +1,300 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+#
+# India holidays en_US localization.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Holidays 0.68\n"
+"POT-Creation-Date: 2025-03-02 12:31+0530\n"
+"PO-Revision-Date: 2025-03-02 12:54+0530\n"
+"Last-Translator: Ankush Kapoor <work.ankushkapoor1626@gmail.com>\n"
+"Language-Team: Holidays Localization Team\n"
+"Language: en_US\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Lingva 5.0.5\n"
+"X-Generator: Poedit 3.5\n"
+"X-Source-Language: en_IN\n"
+
+#. Republic Day.
+msgid "Republic Day"
+msgstr "Republic Day"
+
+#. Independence Day.
+msgid "Independence Day"
+msgstr "Independence Day"
+
+#. Gandhi Jayanti.
+msgid "Gandhi Jayanti"
+msgstr "Gandhi Jayanti"
+
+#. Children's Day.
+msgid "Children's Day"
+msgstr "Children's Day"
+
+#. Labor Day.
+msgid "Labour Day"
+msgstr "Labor Day"
+
+#. Diwali.
+msgid "Diwali"
+msgstr "Diwali"
+
+#. Holi.
+msgid "Holi"
+msgstr "Holi"
+
+#. Govardhan Puja.
+msgid "Govardhan Puja"
+msgstr "Govardhan Puja"
+
+#. Raksha Bandhan.
+msgid "Raksha Bandhan"
+msgstr "Raksha Bandhan"
+
+#. Janmashtami.
+msgid "Janmashtami"
+msgstr "Janmashtami"
+
+#. Dussehra.
+msgid "Dussehra"
+msgstr "Dussehra"
+
+#. Mahavir Jayanti.
+msgid "Mahavir Jayanti"
+msgstr "Mahavir Jayanti"
+
+#. Maha Shivaratri.
+msgid "Maha Shivaratri"
+msgstr "Maha Shivaratri"
+
+#. Navratri / Sharad Navratri.
+msgid "Navratri / Sharad Navratri"
+msgstr "Navratri / Sharad Navratri"
+
+#. Ram Navami.
+msgid "Ram Navami"
+msgstr "Ram Navami"
+
+#. Maha Navami.
+msgid "Maha Navami"
+msgstr "Maha Navami"
+
+#. Ganesh Chaturthi.
+msgid "Ganesh Chaturthi"
+msgstr "Ganesh Chaturthi"
+
+#. Makar Sankranti.
+msgid "Makar Sankranti"
+msgstr "Makar Sankranti"
+
+#. Guru Nanak Jayanti.
+msgid "Guru Nanak Jayanti"
+msgstr "Guru Nanak Jayanti"
+
+#. Ashura.
+msgid "Muharram"
+msgstr "Day of Ashura"
```

Comment:
```suggestion
msgstr "Ashura"
```
Don't listen to bot - we use standardized names of Islamic holidays in en_US. :)